### PR TITLE
feat: add single full report

### PR DESCRIPTION
**Pull request by csiyang**
# Description

- Add /journeys/[journeyId]/reports page
- Embed single full report to the page
- Add button to journey view menu to navigate to the report page

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/27931561/todos/5008377035)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Should show report on /journeys/[journeyId]/reports

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged into main

### CODE REVIEW

File: apps/journeys-admin/src/components/JourneyView/Menu/Menu.spec.tsx [hunk 1]
```diff
@@ -17,9 +18,11 @@ describe('JourneyView/Menu', () => {
     const { getByRole } = render(
       <SnackbarProvider>
         <MockedProvider mocks={[]}>
-          <JourneyProvider value={{ journey: defaultJourney, admin: true }}>
-            <Menu />
-          </JourneyProvider>
+          <FlagsProvider flags={{ analytics: true }}>
```
Comment (tataihono) [inline apps/journeys-admin/src/components/JourneyView/Menu/Menu.spec.tsx:21]:
```suggestion
          <FlagsProvider flags={{ reports: true }}>
```

File: apps/journeys-admin/src/components/JourneyView/Menu/Menu.spec.tsx [hunk 2]
```diff
@@ -51,9 +56,11 @@ describe('JourneyView/Menu', () => {
     const { getByRole } = render(
       <SnackbarProvider>
         <MockedProvider mocks={[]}>
-          <JourneyProvider value={{ journey: publishedJourney, admin: true }}>
-            <Menu />
-          </JourneyProvider>
+          <FlagsProvider flags={{ analytics: true }}>
```
Comment (tataihono) [inline apps/journeys-admin/src/components/JourneyView/Menu/Menu.spec.tsx:59]:
```suggestion
          <FlagsProvider flags={{ reports: true }}>
```

File: apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx [hunk 3]
```diff
@@ -44,6 +46,7 @@ export function Menu({ forceOpen }: MenuProps): ReactElement {
   const [showDescriptionDialog, setShowDescriptionDialog] = useState(false)
   const [showLanguageDialog, setShowLanguageDialog] = useState(false)
   const { enqueueSnackbar } = useSnackbar()
+  const { analytics } = useFlags()
```
Comment (tataihono) [inline apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx:49]:
```suggestion
  const { reports } = useFlags()
```

File: apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx [hunk 4]
```diff
@@ -160,6 +163,16 @@ export function Menu({ forceOpen }: MenuProps): ReactElement {
               </ListItemIcon>
               <ListItemText>Language</ListItemText>
             </MenuItem>
+            {analytics && (
```
Comment (tataihono) [inline apps/journeys-admin/src/components/JourneyView/Menu/Menu.tsx:166]:
```suggestion
            {reports && (
```

**Review by tataihono — DISMISSED** [review]
change all analytics references to reports

File: apps/journeys-admin/src/components/DynamicPowerBiReport/Report/Report.tsx [hunk 5]
```diff
@@ -37,7 +37,7 @@ export function Report({ reportType }: ReportProps): ReactElement {
 
   function onError(): void {
     setError(true)
-    enqueueSnackbar('Error loading Analytics', {
+    enqueueSnackbar('Error loading Reports', {
```
Comment (tataihono) [inline apps/journeys-admin/src/components/DynamicPowerBiReport/Report/Report.tsx:40]:
```suggestion
    enqueueSnackbar('Error loading Report', {
```

File: apps/journeys-admin/src/components/DynamicPowerBiReport/Report/Report.tsx [hunk 6]
```diff
@@ -98,7 +98,7 @@ export function Report({ reportType }: ReportProps): ReactElement {
             <Typography variant="overline" color="secondary.light">
               {error
                 ? 'There was an error loading the report'
-                : 'The analytics are loading...'}
+                : 'The reports are loading...'}
```
Comment (tataihono) [inline apps/journeys-admin/src/components/DynamicPowerBiReport/Report/Report.tsx:101]:
```suggestion
                : 'The report is loading...'}
```

**Review by tataihono — CHANGES_REQUESTED** [review]

**Review by tataihono — DISMISSED** [review]
unsure about decapped `report` in the error message. Happy to approve otherwise.

File: apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.stories.tsx [hunk 7]
```diff
@@ -59,19 +59,19 @@ const Template: Story = ({ ...args }) => {
...
 }
 
-export const Analytics = Template.bind({})
-Analytics.args = {
-  analytics: true,
-  title: 'Analytics'
+export const Reports = Template.bind({})
+Reports.args = {
+  reports: true,
+  title: 'reports'
```
Comment (csiyang) [inline apps/journeys-admin/src/components/PageWrapper/NavigationDrawer/NavigationDrawer.stories.tsx:69]:
```suggestion
  title: 'Reports'
```

File: apps/journeys-admin/src/components/PageWrapper/PageWrapper.stories.tsx [hunk 8]
```diff
@@ -49,19 +49,19 @@ const Template: Story = ({ ...args }) => (
...
+  reports: true
 }
 
-export const Analytics = Template.bind({})
-Analytics.args = {
-  props: { title: 'Analytics' },
-  analytics: true
+export const Reports = Template.bind({})
+Reports.args = {
+  props: { title: 'reports' },
```
Comment (csiyang) [inline apps/journeys-admin/src/components/PageWrapper/PageWrapper.stories.tsx:57]:
```suggestion
  props: { title: 'Reports' },
```

**Review by tataihono — APPROVED** [review]

**Pull request merged**